epick_hardware_tests/src/break.cpp

```diff
@@ -0,0 +1,152 @@
+// on the serial port. It runs an infinite loop of write/read instructions and
+// stops if/when an error is encountered. It usually fails between 1 and 6
+// minutes.
+
+using namespace epick_driver;
+
+constexpr auto kComPort = "/dev/ttyUSB0";
+constexpr auto kBaudRate = 115200;
+constexpr auto kTimeout = 500;  // milliseconds
+constexpr auto kSlaveAddress = 0x09;
+
+int main(int argc, char* argv[])
+{
+  CommandLineUtility cli;
+
+  std::string port = kComPort;
+  cli.registerHandler(
+      "--port", [&port](const char* value) { port = value; }, false);
+
+  int baudrate = kBaudRate;
+  cli.registerHandler(
+      "--baudrate", [&baudrate](const char* value) { baudrate = std::stoi(value); }, false);
+
+  int timeout = kTimeout;
+  cli.registerHandler(
+      "--timeout", [&timeout](const char* value) { timeout = std::stoi(value); }, false);
+
+  int slave_address = kSlaveAddress;
+  cli.registerHandler(
+      "--slave-address", [&slave_address](const char* value) { slave_address = std::stoi(value); }, false);
+
+  cli.registerHandler("-h", [&]() {
+    std::cout << "Usage: ./set_relative_pressure [OPTIONS]\n"
+              << "Options:\n"
+              << "  --port VALUE                 Set the com port (default " << kComPort << ")\n"
+              << "  --baudrate VALUE             Set the baudrate (default " << kBaudRate << "bps)\n"
+              << "  --timeout VALUE              Set the read/write timeout (default " << kTimeout << "ms)\n"
+              << "  --slave-address VALUE        Set the slave address (default " << kSlaveAddress << ")\n"
+              << "  -h                           Show this help message\n";
+    exit(0);
+  });
+
+  if (!cli.parse(argc, argv))
+  {
+    return 1;
+  }
+
+  try
+  {
+    auto serial = std::make_unique<DefaultSerial>();
+    serial->set_port(port);
+    serial->set_baudrate(baudrate);
+    serial->set_timeout(timeout);
+
+    auto driver = std::make_unique<DefaultDriver>(std::move(serial));
+    driver->set_slave_address(slave_address);
+
+    std::cout << "Using the following parameters: " << std::endl;
+    std::cout << " - port: " << port << std::endl;
+    std::cout << " - baudrate: " << baudrate << "bps" << std::endl;
+    std::cout << " - read/write timeut: " << timeout << "ms" << std::endl;
+    std::cout << " - slave address: " << slave_address << std::endl;
+
+    std::cout << "Checking if the gripper is connected..." << std::endl;
+
+    bool connected = driver->connect();
+    if (!connected)
+    {
+      std::cout << "The gripper is not connected" << std::endl;
+      return 1;
+    }
+
+    std::cout << "The gripper is connected." << std::endl;
+    std::cout << "Activating the gripper..." << std::endl;
+
+    driver->activate();
+
+    std::cout << "The gripper is activated." << std::endl;
+    std::cout << "Reading multiple times the gripper status..." << std::endl;
+
+    auto start = std::chrono::steady_clock::now();
```

Comment:
Would it make sense to use "const" here and for all the constant locals below?

Reply:
Done.